fixitfelix/error_handling.py

```diff
@@ -177,19 +178,20 @@ def check_for_correct_repetition(
 
         # extract data points around the data above
         duplicate_front_values = [
-            old_channel.data[min_idx - 1 : min_idx][0]
+            old_channel.read_data(offset=offset-1,length=1)[0]
             for old_channel in all_channels
         ]
         duplicate_rear_values = [
-            old_channel.data[max_idx : max_idx + 1][0]
+            old_channel.read_data(offset=offset+length,length=1)[0]
             for old_channel in all_channels
         ]
+        print(duplicate_front_values, duplicate_rear_values)
```

Comment:
Please remove this debugging print statement

Reply:
Done